### PR TITLE
add angular-ladda w/ npm auto-update 

### LINKED ISSUE #8420: [Request] Add angular-ladda
**Library name:** angular-ladda
**Git repository url:** https://github.com/remotty/angular-ladda
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t Please help me review this PR, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/remotty/angular-ladda
 * Official website (optional, not the repository): https://remotty.github.io/angular-ladda/
 * NPM package url (optional): https://www.npmjs.com/package/angular-ladda
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 210
   - Count of watchers: 16
   - Count of forks: 65
 * NPM download stats (optional):
   - Downloads in the last day: 179
   - Downloads in the last week: 900
   - Downloads in the last month: 2664

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [X] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #8420, cc @subicura

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
There are 27 version tags on git repo but only 13 releases on npm, let's manually add all the versions here.

**Comment by nolsherry** [comment]
@pvnr0082t I have manually added all the versions, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry you should use another commit to do it or mention it in the commit message.

**Comment by nolsherry** [comment]
@pvnr0082t, I have separated to two commits, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry You need to do minify for some unminified files.

**Comment by pvnr0082t** [comment]
Please also mention related issue in first commit message.

**Comment by nolsherry** [comment]
@pvnr0082t , I have minified for some unminified files, mentioned related issue in first commit message and describe why you add old versions in second commit message.
Please help me review this PR, thanks!

**Comment by PeterDaveHello** [comment]
Please do not double close an issue in a single PR next time, thanks.

**Comment by PeterDaveHello** [comment]
Should not generate the map file for the old versions since the new versions released with min files do not contain the map file, should make it consistent.

**Comment by nolsherry** [comment]
@pvnr0082t, I have removed the map files of old versions, please help me review this PR, thanks.

File: ajax/libs/angular-ladda/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "angular",
+    "ladda",
+    "angular-ladda",
+    "ng-ladda"
+  ],
+  "author": "Chungsub Kim <subicura@subicura.com> (http://subicura.com/)",
+  "license": "MIT",
+  "homepage": "https://remotty.github.io/angular-ladda/",
```
Comment (pvnr0082t) [inline ajax/libs/angular-ladda/package.json]: @nolsherry I think it's just a demo page. We can remove it.

Comment (PeterDaveHello) [inline ajax/libs/angular-ladda/package.json]: It looks fine for me.

File: ajax/libs/angular-ladda/0.0.1/angular-ladda.min.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+!function(){"use strict";angular.module("angular-ladda",[]).directive("ladda",["$compile",function(a){return{restrict:"A",link:function(t,n,d){n.html('<span class="ladda-label">'+n.html()+"</span>"),a(n.contents())(t),n.addClass("ladda-button"),n.attr("data-style","zoom-in");var l=Ladda.create(n[0]);t.$watch(d.ladda,function(a){a?l.start():l.stop()})}}}])}();
+//# sourceMappingURL=angular-ladda.min.js.map
```
Comment (PeterDaveHello) [inline ajax/libs/angular-ladda/0.0.1/angular-ladda.min.js]: You need to re-minify the files, there is still  map file info here.

**Pull request closed without merging**